bin/infobot

```diff
@@ -194,28 +190,26 @@ print "Loading Loader: ";
 print "".(($bot->load("Loader"))? "OK" : "NOT OK");
 print "\n";
 
-
-# Then set the keys the plugins
+# Set module parameters from config. Load module if necessary.
 foreach my $key (keys %conf) {
     next if $key eq 'store';
     next unless ref $conf{$key} eq 'HASH';
-    my $mod = $bot->load($key);
-
-
-    unless (defined $mod) {
-        print "We haven't loaded a module $key - config values will be ignored\n";
-        next; # should we 'next' now? bah, wait for people to compain 
+    my $mod;
+    unless ($mod = $bot->module($key)) {
```

Comment:
Wouldn't this be better as my 

    $mod = $bot->module($key) || $bot->load($key);